doc/themes/the-things-stack/assets/css/theme.scss

```
@@ -118,6 +120,14 @@ $tabs-border-bottom-color: $blue;
 @import "bulma/sass/layout/section.sass";
 @import "bulma/sass/layout/footer.sass";
 
+html{
```

```suggestion
html {
```

Please decide on one style to space the brackets.